### PR TITLE
Removed conditional for calling this.props.onChange on iOS (#97)

### PR DESCRIPTION
**Pull request by jacobkring**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -170,10 +170,12 @@ export default class ModalSelector extends React.Component {
     }
 
     onChange = (item) => {
+        this.props.onChange(item);
+        /* this doesn't seem to play nice, the setState below doesn't propagate to the onDismiss state (changeItem) is undefined in the onDismiss function
         if (Platform.OS === 'android' || (Modal.propTypes !== undefined && !Modal.propTypes.onDismiss)) { // don't know if this will work for previous version, please check!
             // RN >= 0.50 on iOS comes with the onDismiss prop for Modal which solves RN issue #10471
             this.props.onChange(item);
-        }
+        }*/
```
Comment (peacechen) [inline index.js]: Please delete the unused code instead of commenting it out.  It will be available in the file history.

File: index.js [hunk 2]
```diff
@@ -313,7 +317,9 @@ export default class ModalSelector extends React.Component {
                 visible={this.state.modalVisible}
                 onRequestClose={this.close}
                 animationType={this.props.animationType}
-                onDismiss={() => this.state.changedItem && this.props.onChange(this.state.changedItem)}
+                onDismiss={() => {
```
Comment (peacechen) [inline index.js]: appreciate the formatting tidiness, but since there's no functional change, please revert this.

File: index.js [hunk 3]
```diff
@@ -313,7 +317,9 @@ export default class ModalSelector extends React.Component {
...
+                    return this.state.changedItem && this.props.onChange(this.state.changedItem)
+                }}
```
Comment (peacechen) [inline index.js]: appreciate the formatting tidiness, but since there's no functional change, please revert this.

File: index.js [hunk 4]
```diff
@@ -303,6 +305,8 @@ export default class ModalSelector extends React.Component {
         );
     }
 
+
+
```
Comment (peacechen) [inline index.js]: Please also revert extra lines.  Try to keep the changes isolated to the functional changes.

**Comment by peacechen** [comment]
Thanks for the PR @jacobkring .   I made some notes above.  Please keep the changes isolated to the specific functional changes, and avoid cosmetic changes in other areas.

File: index.js (hunk 1 above)
Comment (jacobkring) [reply index.js]: Sure thing.

File: index.js (hunk 3 above)
Comment (jacobkring) [reply index.js]: No problem, I think my editor may have done that.

**Pull request closed without merging**